sdk/cloudmachine/Azure.Projects.Provisioning/tests/Data/ofx.bicep

```
@@ -229,13 +184,58 @@ resource cm_eventgrid_subscription_blob 'Microsoft.EventGrid/systemTopics/eventS
 }
 
 resource cm_servicebus_cm0c420d2f21084cd_role 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
-  name: guid(cm_servicebus.id, cm_identity.id, subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39'))
+  name: guid(cm_servicebus.id, project_identity.id, subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39'))
   properties: {
-    principalId: cm_identity.properties.principalId
+    principalId: project_identity.properties.principalId
     roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39')
     principalType: 'ServicePrincipal'
   }
   scope: cm_servicebus
 }
 
-output cm_managed_identity_id string = cm_identity.id
+resource cm_servicebus_subscription_private 'Microsoft.ServiceBus/namespaces/topics/subscriptions@2021-11-01' = {
+  name: 'cm_servicebus_subscription_private'
```

same for the others?

```suggestion
  name: 'project_servicebus_subscription_private'
```

---

I would like to leave for now.